features/presentation-api.yml

```
@@ -1,7 +1,8 @@
-draft_date: 2024-07-11
 name: Presentation API
-description: TODO
+description: The Presentation API allows you to show and control content on another device, such as a network-connected TV.
```

I was not previously aware of this API. "another device" made me think of remote control of another browser (in spec terms, 2-UA). Suggesting a change here to capture the 1-UA sense better:

```suggestion
description: The Presentation API shows or controls content on another display, such as a network-connected TV or projector.
```